package.json

```diff
@@ -7,6 +7,8 @@
     "start": "node src/server.js",
     "dev": "NODE_ENV=development concurrently \" webpack-dev-server --color \" \"nodemon --inspect src/server.js\"",
     "build": "webpack .",
+    "podman:build": " podman image build -t play-reading .",
+    "podman:run": "podman run -v play_volume:/var/lib/postgresql/data --name postgres-db -e POSTGRES_PASSWORD=pass -e POSTGRES_USER=playreading -e POSTGRES_DB=playreadingparty -d -p 5432:5432 play-reading",
```

Comment:
Do you still need to specify the ports? I thought I remember you saying we didn't need to if we weren't changing it

Reply:
Good memory. Fixed.